app/components/initEndpoint/initEndpointController.js

```diff
@@ -26,7 +26,7 @@ function ($scope, $state, EndpointService, StateManager, EndpointProvider, Notif
   $scope.createLocalEndpoint = function() {
     $('#createResourceSpinner').show();
     var name = 'local';
-    var URL = 'unix:///var/run/docker.sock';
+    var URL = 'npipe:\\\\.\\pipe\\docker_engine';
```

Comment:
Of course we won't replace the socket by the npipe when merged. At the moment, this is still acceptable for tests.

Regarding the UX during the first endpoint initialization, we have multiple choices here:

* Detect the platform where Portainer is running and automatically adapt the `URL` value with either the socket path or named pipe
* Replace the two options ('local' and 'remote') by Local (Linux), Local (Windows) and Remote.

Reply:
Agreed, this was just a testing hack - I'll rebase out that commit